src/web/mjs/connectors/ShinobIScans.mjs

```diff
@@ -0,0 +1,14 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
```

Comment:
MAy use `WordPressMadaraNovel` to also support download the [novels](https://shinobiscans.com/manga-genre/novel/) from their website

Reply:
Changed it to MadaraNovel